src/metabase/analyze/fingerprint/fingerprinters.clj

```diff
@@ -58,6 +58,22 @@
    (.offer acc x)
    acc))
 
+(defn- share
+  "Calculate the proportion of inputs for which `pred` returns true. An efficient version of `kixi.stats.core/share`."
```

Comment:
Do you think that maybe it's possible to improve it in kixi.stats? Or is it too specialized for longs?

Reply:
My patch got accepted and released in 0.5.7.